package-lock.json

```diff
@@ -1,8 +1,2143 @@
 {
   "name": "azure-rest-api-specs-tests",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Why is this file being changed?

Reply:
Looks like running prettier validation locally created this. Let me revert this one.

Reply:
Done.